miniapp/USERGUIDE.md

```diff
@@ -46,6 +47,7 @@ The SDK is configured via manifest meta-data, the configurable values are:
 * We don't currently host a public API, so you will need to provide your own Base URL for API requests.
 * All meta-data values must be string values, including the value for `com.rakuten.tech.mobile.miniapp.HostAppVersion`. For example it could be set to the string value `1.0.0`, but if you need to use a number value such as `1.0` or `1`, then you must declare the value in your string resources (`res/values/strings.xml`) and reference the string ID in the manifest, for example `@string/app_version`.
 * The host app info is the string which is appended to user-agent of webview. It should be a meaningful keyword such as host app name to differentiate other host apps.
+* `com.rakuten.tech.mobile.ras.AppId` has been deprecated, use `com.rakuten.tech.mobile.ras.ProjectId` instead.
```

Comment:
Same as my note on iOS: `com.rakuten.tech.mobile.ras.AppId` is still used in other SDKs like Remote Config, so this could be confusing to users of both SDKs. Maybe it's enough to list this in the Changelog?

Reply:
Yes, I agree. I have updated this part.